wasm-node/src/utils.rs

```diff
@@ -0,0 +1,64 @@
+use celestia_node::network;
+use serde_wasm_bindgen::to_value;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+#[derive(PartialEq, Eq, Clone, Copy)]
+pub enum Network {
+    Arabica,
+    Mocha,
+    Private,
+}
+
+#[wasm_bindgen]
+pub fn setup_logging() {
```

Comment:
Instead of requiring user to call this function, we could annotate it with `#[wasm_bindgen(start)]` to get it executed on wasm initialization

Reply:
The bad thing about this, is that user can not set its own tracing logger.

Reply:
Currently set this to be run unconditionally, created https://github.com/eigerco/celestia-node-rs/issues/121 for future tracking.